src/biguint.rs

```diff
@@ -2120,6 +2120,21 @@ impl BigUint {
         }
     }
 
+    /// Returns the u32 digits representation of the `BigUint` in little-endian order.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use num_bigint::BigUint;
+    ///
+    /// let i = BigUint::parse_bytes(b"1125", 10).unwrap();
```

Comment:
`BigUint::from` would be infallible, no `unwrap` needed.
(might need to specify `1125u32` so it doesn't default to `i32`)